src/kakarot/account.cairo

```diff
@@ -439,7 +439,7 @@ namespace Account {
             tempvar range_check_ptr = range_check_ptr;
         }
 
-        tempvar is_not_done = is_le(next_i + 1, bytecode_len);
+        tempvar is_not_done = bytecode_len - next_i;  // if is_not_done == 0, end is reached.
```

Comment:
If the bytecode is malformed this can lead to an overflow (for example you deploy a byecode terminating with a PUSH1 with a CREATE2)

Reply:
Ok
The exit condition is currently `next_i >= bytecode_len`. 
So you confirm a case where `next_i > bytecode_len` is possible ?